pandas/_libs/tslib.pyx

```diff
@@ -211,7 +211,7 @@ def _test_parse_iso8601(object ts):
     if ts == 'now':
         return Timestamp.utcnow()
     elif ts == 'today':
-        return Timestamp.utcnow().normalize()
```

Comment:
side issue, do we really need this actual test routine? can't we just inspect the actual output?

Reply:
Best guess is that this is intended to be double-sure that a test specifically hits string_to_dts (since coverage is hard with cython).  I'd have no real objection to losing this function.

Reply:
yeah i think ok to remove